tests/acceptance/pre-suite/02-install_powershell_module.rb

```diff
@@ -0,0 +1,11 @@
+test_name 'MODULES-3285 - Install Powershell Module'
+
+proj_root = File.expand_path(File.join(File.dirname(__FILE__), '..', '..', '..'))
+staging = { :module_name => 'puppetlabs-powershell' }
+local = { :module_name => 'powershell', :source => proj_root }
+
+hosts.each do |host|
+  step 'Install PowerShell Module'
+  # in CI allow install from staging forge, otherwise from local
+  install_dev_puppet_module_on(host, options[:forge_host] ? staging : local)
+end
```

Comment:
Looks like this file wasn't git moved... and is missing its end of file terminator.